hail/src/main/scala/is/hail/rvd/RVD.scala

```diff
@@ -754,6 +754,14 @@ class RVD(
   ): RVD =
     keyBy(joinKey).orderedJoin(right.keyBy(joinKey), joinType, joiner, joinedType)
 
+  def orderedZipJoin(
```

Comment:
Likewise, this should be moved next to the other `orderedZipJoin`s (a couple methods down).